.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Copyright (c) 2023 Linaro Ltd.
+#
+name: "Builds"
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run periodically to check that it still compiles
+    - cron: '13 13 * * 1'
+  workflow_dispatch:
+
+jobs:
+  job:
+    name: Build
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          - debian:testing
+          - debian:bookworm
+            #- debian:bullseye
+            #- debian:buster
+          - ubuntu:lunar
+          - ubuntu:jammy
+            #- ubuntu:focal
+            #- ubuntu:bionic
+            #- ubuntu:xenial
+
+    container:
+      image: ${{ matrix.container }}
+
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+
+      - name: Install meson
+        run: |
+          apt update
+          apt -y install --no-install-recommends meson build-essential
+
+      - name: Build
+        run: |
+          meson setup . build --werror
+          ninja -C build
+          ninja -C build install
```

Comment:
Shouldn't this cross-compile to `aarch64`?

Reply:
_simple_ CI. I do not want to go into the troubles that we had at CDBA for now. I'd like to flush all the pending PR, by being able to check them before merging. And in the mean time we can implement proper cross-compilation CI.

Reply:
I feel like not compiling to the desired target defeats the purpose, but sure, we'll add it in a followup PR with a proper cross file. Then we can also use `actions/upload-artifact` to provide a prebuilt executable (maybe GNU _and_ musl) from the CI / "release".

Reply:
Added cross-compilation to arm and aarch64 architectures. We don't have debugcc for 32-bit arm platforms, but maybe it will come one day.